python/tvm/relay/frontend/onnx.py

```diff
@@ -961,6 +961,18 @@ def _impl_v1(cls, inputs, attr, params):
         return _op.power(out, reci_p)
 
 
+class GlobalLpPool(OnnxOpConverter):
+    """Operator converter for GlobalLpPool."""
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
+        in_shape = infer_shape(inputs[0])
+        assert len(in_shape) >= 4, "input shape support NCHW or NCDHW"
```

Comment:
LpPool supports the 1D case e.g. N x C x D_1 and the onnx spec makes it seems like we should support it too.